sergii.holubenko/task_1.rb

```diff
@@ -0,0 +1,25 @@
+arr = [a, b, c]
+a = []
+3.times do
```

Comment:
if you have block that contains only one line it could written like this: `3.times { arr.push(rand(-10...10)) }`